### ISSUE TITLE
YAML dependency error

### ISSUE THREAD

**Issue by OrianeN** [patacrep/patacrep#274]
PyYAML>= 6.0.0 now requires positional argument 'Loader' when calling function yaml.load(). cf. https://github.com/yaml/pyyaml/commit/c2743653bc89e42679ba097b4f9888db47c61d63

Therefore, when running `songbook example.yaml`, I've encountered the following error:

> ERROR:root:load() missing 1 required positional argument: 'Loader'

PyYAML documentation at https://pyyaml.org/wiki/PyYAMLDocumentation suggests the following:
```python
from yaml import load, dump
try:
    from yaml import CLoader as Loader, CDumper as Dumper
except ImportError:
    from yaml import Loader, Dumper

data = load(stream, Loader=Loader
```

**Comment by Grahack** [comment]
If ever the fix works for you don't hesitate to make a PR.

**Comment by Flore-mozfr** [comment]
Bonsoir,
Ca marche bien avec safe_load

**Comment by sgelis** [comment]
Fixed by #269.
